src/python/foglamp/rest/util.py

```diff
@@ -1,9 +1,9 @@
 '''Defines utility function used
 by all uri handlers'''
 
-import json
 from datetime import datetime
 from aiohttp import web
+import json
 
 def json_response(body='', **kwargs):
```

Comment:
This change is not of this PR, but just to bring it to attention:

`web.json_response(DATA_TO_SEND)` exists in aiohttp.